sync.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# x86_64 or arm64
+[ "$(uname -m)" != "x86_64" ] && USE_ARM64="-arm64"
+
+# security or no security
+[ "$SECURITY_SERVICE_NEEDED" != true ] && USE_NO_SECURITY="-no-secty"
+
+# redis or mongo
+[ "$FOR_REDIS" = true ] && USE_REDIS="-redis"
+
+# nightly or other release
+USE_RELEASE=${RELEASE:-nightly-build}
+if [ "$USE_RELEASE" = "nightly-build" ]; then
+     export COMPOSE_FILE="releases/${USE_RELEASE}/compose-files/docker-compose-nexus${USE_REDIS}${USE_NO_SECURITY}${USE_ARM64}.yml"
+else
+     export COMPOSE_FILE="releases/${USE_RELEASE}/compose-files/docker-compose${USE_REDIS}-${USE_RELEASE}${USE_NO_SECURITY}${USE_ARM64}.yml"
+fi
+
+echo $COMPOSE_FILE
+git remote add dev https://github.com/edgexfoundry/developer-scripts.git
+git checkout -b devBranch
+git fetch dev
+git checkout dev/master -- $COMPOSE_FILE
```

Comment:
Recommend that when `uname -m` = "x86_64" to apply the following transformation

```
sed -i 's/kong:1.3.0$/kong:1.3.0-centos/' $COMPOSE_FILE
```

Recommend that when `uname -m` = "aarch64" to apply the following transformation

```
sed -i 's/kong:1.3.0$/kong:1.3.0-ubuntu/' $COMPOSE_FILE
```

Reply:
Why isn't the kong image changed in developer-script instead? This PR [edgexfoundry/developer-scripts#187](https://github.com/edgexfoundry/developer-scripts/pull/187/files) will add docker-compose files which contain kong:1.3.0-ubuntu for arm64.

Reply:
> Why isn't the kong image changed in developer-script instead? This PR [edgexfoundry/developer-scripts#187](https://github.com/edgexfoundry/developer-scripts/pull/187/files) will add docker-compose files which contain kong:1.3.0-ubuntu for arm64.

Good point.

How about this then:

echo $COMPOSE_FILE
wget -q -O docker-compose.yml "https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/${COMPOSE_FILE}"

# Use Centos base image instead of Alpine base image for Kong for x86_64 CI
# due to compatibility issues with Alpine image in CI
if [ `uname -m` = "x86_64" ]; then
    sed -i 's/kong:1.3.0$/kong:1.3.0-centos/' docker-compose.yml
fi

Reply:
Changed to:
```
wget -q "https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/releases/${USE_RELEASE}/compose-files/${COMPOSE_FILE}"

# Use Centos base image instead of Alpine base image for Kong for x86_64 CI
# due to compatibility issues with Alpine image in CI
# sed command of MacOS is in different syntax 
[ "$(uname -m)" = "x86_64" ] && [ "$(uname)" = "Darwin" ] && sed -i '' 's/kong:1.3.0$/kong:1.3.0-centos/' ${COMPOSE_FILE} || sed -i 's/kong:1.3.0$/kong:1.3.0-centos/' ${COMPOSE_FILE}

```